modules/lib/maintainers.nix

```diff
@@ -41,4 +41,10 @@
     github = "olmokramer";
     githubId = 3612514;
   };
+  colemickens = {
+    name = "Cole Mickens";
+    email = "cole.mickens@gmail.com";
+    github = "colemickens";
+    githubId = 327028;
+  };
```

Comment:
AFAIK this list is for maintainers that are not in Nixpkgs `maintainers/maintainer-list.nix` yet.